Pear/Controller/HomeViewController.swift

```diff
@@ -30,6 +30,9 @@ class HomeViewController: UIViewController {
     private var displayMenu = true
     private let profileButtonSize = CGSize(width: 35, height: 35)
     private let tabs = ["Weekly Pear", "People"]
+    private var menuOptions = ["Send feedback", "Contact us", "Report user"]
```

Comment:
similar to the comment about factoring out images, maybe do something similar with strings. I'm seeing that a lot of these strings are duplicated. 

you shouldn't worry about making these changes in this PR, bring it up to mat + noah and they can be a separate PR :)